src/oc_erchef/rebar.config

```diff
@@ -60,17 +59,19 @@
         {darklaunch, ".*",
          {git, "https://github.com/chef/opscode-darklaunch-erlang", {branch, "master"}}},
         {opscoderl_httpc, ".*",
-         {git, "https://github.com/chef/opscoderl_httpc", {branch, "master"}}},
+         {git, "https://github.com/chef/opscoderl_httpc", {branch, "ma/ibrowse_uses_mainline"}}},
         {sync, ".*",
          {git, "https://github.com/rustyio/sync.git", {branch, "master"}}},
         {prometheus, ".*",
          {git, "https://github.com/deadtrickster/prometheus.erl", {tag, "v3.4.0"}}},
+        {eper, ".*",
+         {git, "https://github.com/massemanet/eper", {branch, "master"}}},
         {observer_cli, ".*",
```

Comment:
Any reason for changing the order here?

Reply:
I forgot why.  However, all the `rebar.config`s were redone in alphabetical order in a later commit.